scripts/install-chef-suse.sh

```diff
@@ -278,6 +278,15 @@ post_fail_handler ()
     rm -f $crowbar_install_dir/crowbar_installing
 }
 
+function rails_db_cleanup()
+{
+    rm -f $crowbar_install_dir/crowbar-install-failed
+    rm -f $installation_steps
```

Comment:
So these two bits are not really related to db cleanup, so maybe rename the function (reset_crowbar, maybe -- not super good name either, though)?